code/game/machinery/lightswitch.dm

```diff
@@ -47,7 +47,7 @@
 		update_icon(UPDATE_ICON_STATE)
 
 /obj/machinery/light_switch/Initialize()
-	..()
+	. = ..()
 	set_frequency(frequency)
 	name = "light switch"
```

Comment:
Why the bloody hell is this here?
*also good god I just looked at the above proc*

Reply:
I claim "out of scope" on this